raphtory/src/db/api/view/graph.rs

```diff
@@ -152,12 +152,19 @@ impl<'graph, G: BoxableGraphView<'graph> + Sized + Clone + 'graph> GraphViewOps<
         }
 
         for v in self.nodes().iter() {
+            let v_type_string = v.node_type(); //.map(|v| v.to_string()); //stop it being dropped
+            let v_type_str = v_type_string.as_str();
```

Comment:
If we make `node_type()` return `Option<&ArcStr>` we wouldn't have to do this dance.